examples/C/src/rosace/RosaceWithUIandWatchdog.lf

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
nit: Maybe we should name this RedundantRosaceWithWatchdog.lf?

Reply:
Done.